app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -0,0 +1,167 @@
+import { Hidden, makeStyles, Paper } from "@material-ui/core";
+import Alert from "components/Alert";
+import CircularProgress from "components/CircularProgress";
+import HorizontalScroller from "components/HorizontalScroller";
+import TextBody from "components/TextBody";
+import SearchBox from "features/search/SearchBox";
+import SearchResult from "features/search/SearchResult";
+import { Error } from "grpc-web";
+import { LngLatBounds, Map as MaplibreMap } from "maplibre-gl";
+import { User } from "pb/api_pb";
+import { UserSearchRes } from "pb/search_pb";
+import { searchQueryKey } from "queryKeys";
+import React, { MutableRefObject } from "react";
+import { useInfiniteQuery } from "react-query";
+import { useParams } from "react-router-dom";
+import { service } from "service";
+import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
+
+const useStyles = makeStyles((theme) => ({
+  mapResults: {
+    height: "14rem",
+    zIndex: 3,
+    overflow: "hidden",
+    [theme.breakpoints.up("md")]: {
+      height: "auto",
+      width: "35%",
+      overflow: "auto",
+    },
+  },
+  noResults: { margin: theme.spacing(2) },
+  searchDesktop: {
+    margin: theme.spacing(0, 2),
+  },
+  scroller: {
+    "&&": { alignItems: "flex-start" },
+  },
+  searchResult: {
+    [theme.breakpoints.down("sm")]: {
+      maxWidth: "80%",
+    },
+    [theme.breakpoints.up("md")]: {
+      margin: theme.spacing(2),
+    },
+  },
+}));
+
+interface SearchResultsListProps {
+  handleResultClick(user: User.AsObject): void;
+  map: MutableRefObject<MaplibreMap | undefined>;
+  selectedResult?: number;
+}
+
+export default function SearchResultsList({
```

Comment:
I think this is broken for tablet breakpoint that's not quite at desktop width, as I see this and can't scroll down until it becomes small enough to switch to the horizontal scroller:
![Screenshot 2021-04-26 at 00 10 38](https://user-images.githubusercontent.com/13669362/116012680-ecb0ad80-a623-11eb-9aa2-b4a0334e8232.png)

Reply:
This is still a problem. Maybe we should show this as the horizontal scroller at medium breakpoint too? Seems like the easiest fix and I don't think it'd look too bad